pkg/backend/filestate/backend_test.go

```diff
@@ -471,3 +472,122 @@ func TestHtmlEscaping(t *testing.T) {
 	state := string(bytes)
 	assert.Contains(t, state, "<html@tags>")
 }
+
```

Comment:
This is all from project support for filestate, think you've got a bad merge adding it here.

Reply:
Removed!